concepts/05 UI Components/Gantt/20 Filter Data/1 Filter Row.md

```diff
@@ -0,0 +1,407 @@
+The **Gantt** component allows users to filter columns by filter row values. To apply filter criteria, users enter a value in a column's filter row cell or select the value with a specific editor. 
+
+Set the **filterRow**.[visible](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/filterRow/#visible) property to **true** to display the filter row. Assign **false** to the **columns.**[allowFiltering](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#allowFiltering) property to disable the filtering feature for a column. 
+
+![DevExtreme HTML5 JavaScript jQuery Knockout Angular Gantt Filtering FilterRow](/images/Gantt/Visual_elements/Filter_row.png)
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/FilterRow/"
+}
+
+The filter row contains a magnifier icon in its cells. Users click this icon to select the desired filter operation. Set the [showOperationChooser](/Documentation/21_2/ApiReference/UI_Components/dxGantt/Configuration/filterRow/#showOperationChooser) property to **false** for a column to hide the magnifier icon. 
```

Comment:
What do you mean by "desired filter operation"? Does a click on the magnifier icon invokes a pop-up menu that allows one to choose between available options? I believe we need more clarity about this here.

```suggestion
The filter row contains a magnifier icon in its cells. Users can click this icon to select the desired filter operation. Set the [showOperationChooser](/Documentation/21_2/ApiReference/UI_Components/dxGantt/Configuration/filterRow/#showOperationChooser) property to **false** for a column to hide the magnifier icon. 
```

Reply:
upd:
```suggestion
The filter row contains a magnifier icon in its cells. Users click this icon to select a comparison type from a pop-up list. Set the [showOperationChooser](/Documentation/21_2/ApiReference/UI_Components/dxGantt/Configuration/filterRow/#showOperationChooser) property to **false** for a column to hide the magnifier icon. 
```

Reply:
Looks good to me.